list-pr-files/src/index.js

```diff
@@ -0,0 +1,27 @@
+const core = require("@actions/core");
+const github = require("@actions/github");
+
+async function main() {
+  try {
+    const token = core.getInput("ghToken");
```

Comment:
```suggestion
    const token = core.getInput("gitHubToken");
```

Reply:
Is it possible that the input is not set? If so, what would the error message be the user sees?

Reply:
No, the input is defined as required in `action.yml`. I don't know the exact error they'll see but it's the standard GHA missing input error message.

Reply:
Correction: it's not required anymore but the default will cover most cases.